<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrashDialog</class>
 <widget class="QDialog" name="CrashDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crash report</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="InfoLabel_">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="TraceDisplay_"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>You can save the report for later or just accept this dialog, and in this case the Dolozhee module (if installed) will prompt you to upload the crash report to our bugtracker next time LeechCraft starts.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Reload_">
       <property name="text">
        <string>Reload symbols</string>
       </property>
       <property name="icon">
        <iconset theme="view-refresh">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Copy_">
       <property name="text">
        <string>Copy</string>
       </property>
       <property name="icon">
        <iconset theme="edit-copy">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Save_">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset theme="document-save">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="RestartBox_">
     <property name="text">
      <string>Restart</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="DialogButtons_">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>DialogButtons_</sender>
   <signal>rejected()</signal>
   <receiver>CrashDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>916</x>
     <y>600</y>
    </hint>
    <hint type="destinationlabel">
     <x>1014</x>
     <y>581</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DialogButtons_</sender>
   <signal>accepted()</signal>
   <receiver>CrashDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>883</x>
     <y>598</y>
    </hint>
    <hint type="destinationlabel">
     <x>1009</x>
     <y>556</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
